### PR TITLE
Update certificate interface as webIDE-common-tizentv

### PR DESCRIPTION
**Pull request by keyuan0701**

### CODE REVIEW

**Review by jianhua-git — COMMENTED** [review]
checked

**Review by fjk0401 — COMMENTED** [review]
Good  for me

File: lib/samsungCertificate.js [hunk 1]
```diff
@@ -180,7 +180,15 @@ var samsungCertificate = (function() {
 
                     // Generate Author certificate
                     authorCreatePage.getView().loadingStart();
-                    samsungCertMgr.ceateAuthorCert(profileName, authorInfo, accessInfo).then((message) => {
+                    samsungCertMgr.createAuthorCert(profileName, {
```
Comment (prozanne) [inline lib/samsungCertificate.js:183]: This is not related with this line.
But I found a type on "samsungCertificateAuthorCreateView.js", "samsungCertificateDistributorCreateView.js", "tizenCertificateAuthorCreateView.js"

passoword => password

File: lib/tizenCertificate.js [hunk 2]
```diff
@@ -196,7 +197,17 @@ var tizenCertificate = (function(){
 				try {
 					// Handle author certificate information
 					if (certInfo.authorInfo.type == 'create') {
-						tizenCertMgr.createCert(certInfo.authorInfo.info);
+						tizenCertMgr.createCert({
+							keyfileName: certInfo.authorInfo.info.authorFile,
```
Comment (prozanne) [inline lib/tizenCertificate.js:201]: The property names were changed.

File: lib/samsungCertificate.js (hunk 1 above)
Comment (keyuan0701) [reply lib/samsungCertificate.js:183]: done

File: lib/tizenCertificate.js (hunk 2 above)
Comment (keyuan0701) [reply lib/tizenCertificate.js]: done

File: lib/tizenCertificate.js [hunk 3]
```diff
@@ -196,7 +197,17 @@ var tizenCertificate = (function(){
 				try {
 					// Handle author certificate information
 					if (certInfo.authorInfo.type == 'create') {
-						tizenCertMgr.createCert(certInfo.authorInfo.info);
+						tizenCertMgr.createCert({
```
Comment (prozanne) [inline lib/tizenCertificate.js:200]: At the end of the step for creating tizen certification, I saw success popup with check box which mentions "set active profile", it seems default value is false, but I think "true" is much better as usability aspect.

File: lib/witsLauncher.js [hunk 4]
```diff
@@ -27,20 +27,13 @@ async function handleCommand(option) {
...
     let appWidth = getAppScreenWidth(workspacePath);
-
     let debugMode = false;
     let connection = await checkTVConnection(debugMode);
     if (connection == undefined) {
-        logger.error(moduleName, 'connection is undefined. Please check the host and TV IP');
-        common.showMsgOnWindow(common.ENUM_WINMSG_LEVEL.WARNING, 'Please check the host and TV IP!');
+        let errMsg = 'connection is undefined. Please check the host and TV IP: Packages > Setting view > Open > Packages > search [atom-tizentv] > Settings';
+        logger.error(moduleName, errMsg);
+        common.showMsgOnWindow(common.ENUM_WINMSG_LEVEL.WARNING, errMsg);
```
Comment (prozanne) [inline lib/witsLauncher.js:36]: From 61 to 84 lines, It can't be here. If there is issue about tools permission, it should be implemented on WITs, not here.

Comment (prozanne) [inline lib/witsLauncher.js:36]: Also 41 line, 
proxyServer setting is not required anymore.
Please remove them.

41 line and 49 line

Comment (keyuan0701) [reply lib/witsLauncher.js:36]: done

Comment (keyuan0701) [reply lib/witsLauncher.js:36]: done

File: lib/tizenCertificate.js (hunk 3 above)
Comment (keyuan0701) [reply lib/tizenCertificate.js:200]: done

File: package.json [hunk 5]
```diff
@@ -15,8 +15,8 @@
     "atom": ">=1.0.0 <2.0.0"
   },
   "dependencies": {
-    "@tizentv/wits": "^2.1.5",
-    "@tizentv/webide-common-tizentv": "0.0.1",
+    "@tizentv/wits": "^2.2.1",
+    "@tizentv/webide-common-tizentv": "0.0.3",
```
Comment (prozanne) [inline package.json:19]: Pull request is progressing that @tizentv/webide-common-tizentv's version up(v1.0.0).

Comment (keyuan0701) [reply package.json]: done

**Review by prozanne — APPROVED** [review]
checked

**Review by chunhua-liu — APPROVED** [review]

**Review by prozanne — APPROVED** [review]
